authorization-common/src/main/resources/legacyRights.json

```diff
@@ -78,5 +78,6 @@
   "datahub/read": "datahub/read",
   "datahub/write": "datahub/write",
   "structured_events/read": "structured_events/read",
-  "iam/listAssignedResourceRoles": "iam/listAssignedResourceRoles"
+  "iam/listAssignedResourceRoles": "iam/listAssignedResourceRoles",
+  "environments/getAuditEvents": "environments/describeEnvironment"
```

Comment:
There is no legacy right like this... please read the manual because I think you haven't read it yet:
https://cloudera.atlassian.net/wiki/spaces/ENG/pages/689636257/Resource+based+authorization+in+CB+not+enabled+yet
https://github.com/hortonworks/cloudbreak/blob/master/authorization-common/README.md